oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/observation/EventFactory.java

```diff
@@ -242,18 +242,18 @@ private abstract class EventImpl implements JackrabbitEvent {
         /**
          * Path of the parent node of the item this event is about.
          */
-        private final String parent;
+        private final String parentPath;
 
         /**
          * Name of the item this event is about.
          */
-        protected final String name;
+        protected final String itemPath;
```

Comment:
This contradicts the description in the javadoc.

Reply:
Thx